Presentation.Web/app/models/ui-customization/config/blueprints/it-system-usage-ui-customization-config-blueprint.ts

```diff
@@ -16,6 +16,9 @@
             systemRelations: {
                 text: "Relationer"
             },
+            dataProcessings: {
```

Comment:
dataProcessing (not the s)

Reply:
must be read only

Reply:
and use the same help text as contracts and projects
![image](https://user-images.githubusercontent.com/13116889/177776618-df2b1bef-0057-4e76-aba0-b6478fc31308.png)